TozSdk/TozSdk.csproj

```diff
@@ -4,11 +4,27 @@
     <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+
+    <!-- NuGet Package Metadata -->
+    <PackageId>Tozny.CSharp.Sdk</PackageId>
+    <Version>1.0.1</Version>

```

Comment:
in the proccess cna u add upadte this version

Reply:
Done!